src/style.css

```diff
@@ -1 +1,38 @@
 /* styles go here */
+
+.infoAbout1Form {
+  margin-bottom: 10px;
+}
+
+input[type="checkbox"] {
+  position: relative;
+  top: 2px;
+}
+
+.infoAboutRegistr {
+  padding-top: 10px;
+}
+
+.interestFact {
+  margin-bottom: 10px;
+}
+
+.additionalInfo {
+  margin-bottom: 10px;
+}
```

Comment:
why do you need to create separate class for every element? just create generic class and reuse it